airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -372,6 +379,32 @@ def get_num_queued_tasks():
             .count()
         )
 
+def insert_latest_successful_run(session, insert_dict):
+    for key, execution_date in insert_dict.items():
+        dag_id, task_id = key
+        session.add(LatestSuccessfulRun(
+            dag_id=dag_id,
+            task_id=task_id,
+            execution_date=execution_date,
+        ))
+    session.flush()
+    session.commit()
+
+def update_latest_successful_run(session, update_dict):
```

Comment:
since you are passing a dict, maybe rename this to `update_latest_successful_runs`?